.github/ISSUE_TEMPLATE/content.md

```diff
@@ -13,17 +12,16 @@ Thank you for wanting to make amp.dev better.
 Please fill in as much of the below template as you're able.
 -->
 
-## 📖 Missing or out-of-date documentation
+## 📖 Missing or Out-of-Date Documentation
 
-**Describe the content that is missing or should be up dated.**  
-A clear and concise description of what content is missing, or needs to be updated.
-Could be a missing guide explaining a concept, a tutorial, or something that is confusing or doesn't work when you tried it.
+### Problem
+A clear and concise description of what content is missing or needs updating. It could be a missing guide explaining a concept, a step-by-step tutorial, something that is confusing or didn't work when you tried it.
```

Comment:
I'm still not pleased with this paragraph because I don't know if it should be

> It could be a missing guide explaining a concept, a step-by-step tutorial, something that is confusing or didn't work when you tried it.

(as it currently is)

or 

> It could be a missing guide explaining a concept, a step-by-step tutorial, something that is confusing, _or something_ that didn't work when you tried it.

Reply:
Preferring Oxford comma. Fixed in commit [27f2a0f](https://github.com/ampproject/amp.dev/pull/3326/commits/27f2a0f4dc1a8c41aa832af42bc8500d9215b015).